### ISSUE TITLE
Obsolete: methotrexate terms

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#21998]
The proposal has been made to obsolete all the methotrexate terms for the reason given below.

This was discussed at the GO annotation call on 9/7/2021.

* **GO term ID and Label**
- GO:0051870 - methotrexate binding (F)
- GO:0015350 - methotrexate transmembrane transporter activity (F)
- GO:0051958 - methotrexate transport (P)
- GO:0031427 - response to methotrexate (P)

* **Reason for deprecation**
The reason for obsoletion is that methotrexate is a synthetic compound that is an analog of folic acid. It's effects on organisms are usually due to its similarity to folic acid, not due to any organism being evolved to deal with methotrexate specifically.

* **"Replace by" term (ID and label)**
If all annotations can safely be moved to that term

NA

* **"Consider" term(s) (ID and label)**
Suggestions for reannotation

- GO:0051870 - methotrexate binding (F)
=> consider: 
-- GO:0005542 - folic acid binding 
-- DELETING annotation

- GO:0015350 - methotrexate transmembrane transporter activity (F)
=> consider: 
-- GO:0008517 - folic acid transmembrane transporter 
-- GO:0015347 - sodium-independent organic anion transmembrane transporter activity (make new Na-ind polyspecific term?? or make this term polyspecific??)
-- DELETING annotation

- GO:0051958 - methotrexate transport (P)
=> consider: 
-- GO:0015884 - folic acid transport 
-- DELETING annotation

- GO:0031427 - response to methotrexate (P)
=> consider: 
-- GO:new - negative regulation of dihydrofolate reductase activity
-- DELETING annotation

* **Are there annotations to this term?**
- How many EXP: 32 experimental annotations by ARUK-UCL, BHF-UCL, CAFA, RGD, or UniProt
- GeneDB & MGI only have annotations by ISS or ISO
- Ensembl, GOC, and ZFIN only have annotations by IEA

<img width="665" alt="methotrexateAnnotsByEvCode" src="https://user-images.githubusercontent.com/10533218/130148292-9dadf0fb-6461-4f70-bac5-a5df1e1160b7.png">

* **Are there mappings and cross references to this term? (InterPro, Keywords; check QuickGO cross-references section)**

no

* **Is this term in a subset? (check the AmiGO page for that term)**

no

* **Any other information**
Most of the genes annotated to the methotrexate terms appear to be involved in folic acid metabolism or transport (shown in grean). One gene, Slco1a3 (shown in red), is a polyspecific anion transporter that can transport methotrexate as well as other organic anions.

<img width="319" alt="genesAnnotdToMTXterms" src="https://user-images.githubusercontent.com/10533218/130190145-82f66728-ea8f-4fbf-b4a1-060a615d48a3.png">

**Comment by paolaroncaglia** [comment]
Hi @krchristie and GO editors,

The term 
GO:0031427 - response to methotrexate (P)
is imported in the Experimental Factor Ontology (EFO) and used by curators at the Genome-Wide Association Studies (GWAS) catalog. EFO imports many 'response to drug' terms, as you may see expanding the node [here](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0042493&viewMode=All&siblings=false). As GO editors we addressed this issue 5 years ago in [this ticket](https://github.com/geneontology/go-ontology/issues/12228). In summary, at the time we resolved not to obsolete such terms, but rather place them in the do_not_manually_annotate subset, and add a definition comment "Note that this term is in the subset of terms that should not be used for direct manual annotation of gene products. It was created to be used for cross-referencing by other ontologies. Direct annotations to this term may be amended during annotation QC.". I do appreciate that 'response to methotrexate' should not be used to annotate when it's used as a proxy for folic acid, but the intent of GWAS curators is different (see [here](https://www.ebi.ac.uk/gwas/efotraits/GO_0031427) for the use of the term in the GWAS catralog). Would you please consider to keep GO:0031427, place it in the do_not_manually_annotate subset, and add a definition comment as above?

I'm tagging my EFO colleagues @zoependlington and @kallia-p :
- [ ] If GO still wish to go ahead and obsolete GO:0031427 'response to methotrexate', the alternative option for EFO would be to mint a new EFO term with the same meaning, use it to replace the GO term, and alert GWAS curators
- [ ] It may be helpful to revise what children of 'response to drug' are currently imported in EFO as GO terms and used by GWAS. If they don't have a usage restriction and def. comment as above (which you'll need to verify using the QuickGO browser, e.g. [here](https://www.ebi.ac.uk/QuickGO/term/GO:1902521), because OLS don't display those external properties), you may want to liaise with GO to prevent future obsoletion.

Thanks,
Paola

**Comment by krchristie** [comment]
Hi @paolaroncaglia @kallia-p @zoependlington @dosumis 

We can **_definitely_** delay obsoletion until a plan is in place to make sure that GWAS curators are not disrupted. 

Thanks @paolaroncaglia for pointing out the old ticket on **Response to compounds used as assays #12228** . That is very helpful. Keeping these terms in GO certainly made sense 5 years ago. At this point in time though, if EFO can accomodate these terms, I think that would be a better place for them than GO, and better than keeping a somewhat haphazard set of these kinds of terms in GO marked in the do_not_manually_annotate subset. 

Another specific term that has come up recently is "response to paclitaxel", see the **response to paclitaxel #21750**  ticket.

GO is also planning to merge the term "response to drug" into the term "response to "xenobiotic", see this ticket:  **handling terms for xenobiotic role and removing drug role #19460** 

As GO is moving towards removing these kinds of terms, and EFO is now a good place for them, it seems like it might be a good time to come up with a plan to move forwards with EFO taking over terms like this that really don't belong in GO. 

@pgaudet what do you think about this idea? Maybe at some point, it would be good to see if EFO curators can attend our GO editors meeting (currently Mondays at 9 am Pacific, 5 pm UK), if that time would work for you.

**Comment by paolaroncaglia** [comment]
Hi @krchristie ,

Thank you very much for your reply.
@kallia-p has outlined a detailed plan [here](https://github.com/EBISPOT/efo/issues/32#issuecomment-925151091). Based on that, I _think_ that there may be no need for an urgent EFO-GO meeting, but I'll leave this to Kallia, as I'm not involved in GWAS work at the moment.  

Wishing you a great day,
Paola

**Comment by kallia-p** [comment]
Hi @krchristie @paolaroncaglia @zoependlington
Thank you for informing us of your proposed changes and for explaining the rationale behind these changes. EFO users such as the GWAS catalog and Open Targets would prefer to cross-reference their pharmacogenomic response terms to GO biological processes and therefore any changes to the broad parental classes such as the merge of 'response to drug' with the 'response to xenobiotic' are more important to our users than the children of this branch for example, response to paclitaxel. Since @zoependlington is the contact person for OTAR, I will liaise with her to discuss ways of tackling these proposed changes. 
Thanks, Kallia

**Comment by kallia-p** [comment]
Hi @krchristie @zoependlington @paolaroncaglia 

I think, given the timing it would be best if we could both approach the 'response to' terms in a systematic way rather than as part of sporadic tickets about specific drugs. If you agree we could start discussing this in a new ticket. If needed I can join one of your future meetings to discuss and plan around the proposed work with regards to the alignment between GO, EFO and OBA 'response to' terms. Please let me know which of the two you prefer and if you think a meeting is warranted which forthcoming Monday we can allocate to this task. 

I provide a bit of background and context to the preliminary work that I have done thus far.
I am currently reviewing all of the terms that are in GO and have been imported in EFO (N=75 GO terms, see table below which comprises the EFO created 'response to' specific drug terms as well).  I am also examining the OBA cross referenced terms with GO and the overlap between the three ontologies. With regards to EFO some of the imported GO terms are high level grouping terms with the main one being 'response to stimulus' of which 'response to chemical' is a Subclass Of. 'Response to drug' is a Subclass Of 'response to chemical' and I have made a note that you plan to merge this with 'response to xenobiotic'. Further down the GO hierarchy i.e. in the 'response to drug' branch there is some structure incorporated in GO that several EFO terms sit under for example the GO 'response to antipsychotic drug' and GO 'response to antineoplastic agent' have a number of children with either GO or EFO IDs for specific antipsychotic drugs and cancer therapy agents (with one of them being the GO 'response to paclitaxel' which you are planning to obsolete). Small changes like these will have less of an effect to our users but changes in higher level grouping terms will cause much disruption and I wonder why do you think EFO would be a better place for them than GO, and what is the hazard of keeping these kind of terms in GO.

I reviewed the tickets in trying to understand how GO views usage of the 'response to' terms in general. 
In my view an organism's response to an exogenous stimulus whether this is food intake, stress or a pharmacologic intervention constitutes a biological process; variations in our our genes interact with our environment. Many of the GWAS catalog studies associate variants with phenotypes that measure responses.  Pharmacogenomics studies will specifically look for variants that modulate efficacy of a drug or adverse/toxic reactions. The GWAS catalog contains associations from gene x environment interactions studies as well. I think it also useful for researchers to know that (a variant affecting expression or function of) gene X which is associated with an adverse reaction to a drug Y (GO 'response to term') is linked to GO biological process Z. 

Thanks!

<google-sheets-html-origin><style type="text/css"></style>

subject_id | subject_label
-- | --
GO:1990054 | response to temozolomide
GO:1905119 | response to haloperidol
GO:1904014 | response to serotonin
GO:1903493 | response to clopidogrel
GO:1903492 | response to aspirin
GO:1903491 | response to simvastatin
GO:1902522 | response to epirubicin
GO:1902521 | response to etoposide
GO:1902520 | response to doxorubicin
GO:1902519 | response to docetaxel
GO:1902518 | response to cyclophosphamide
GO:1901905 | response to tamsulosin
GO:1901563 | response to camptothecin
GO:1901559 | response to ribavirin
GO:1901558 | response to metformin
GO:1901557 | response to fenofibrate
GO:1901556 | response to candesartan
GO:1901555 | response to paclitaxel
GO:1901554 | response to acetaminophen
GO:0097366 | response to bronchodilator
GO:0097338 | response to clozapine
GO:0097337 | response to ziprasidone
GO:0097336 | response to risperidone
GO:0097335 | response to quetiapine
GO:0097334 | response to perphenazine
GO:0097333 | response to olanzapine
GO:0097332 | response to antipsychotic drug
GO:0097331 | response to cytarabine
GO:0097330 | response to 5-fluoro-2'-deoxyuridine
GO:0097329 | response to antimetabolite
GO:0097328 | response to carboplatin
GO:0097327 | response to antineoplastic agent
GO:0072758 | response to topoisomerase inhibitor
GO:0072718 | response to cisplatin
GO:0071467 | cellular response to pH
GO:0071214 | cellular response to abiotic stimulus
GO:0061479 | response to reverse transcriptase inhibitor
GO:0061478 | response to platelet aggregation inhibitor
GO:0061477 | response to aromatase inhibitor
GO:0061476 | response to anticoagulant
GO:0051716 | cellular response to stimulus
GO:0051384 | response to glucocorticoid
GO:0050896 | response to stimulus
GO:0048583 | regulation of response to stimulus
GO:0046677 | response to antibiotic
GO:0043278 | response to morphine
GO:0042630 | behavioral response to water deprivation
GO:0042493 | response to drug
GO:0036288 | response to ximelagatran
GO:0036287 | response to iloperidone
GO:0036277 | response to anticonvulsant
GO:0036276 | response to antidepressant
GO:0036275 | response to 5' fluorouracil
GO:0036274 | response to lapatinib
GO:0036273 | response to statin
GO:0036272 | response to gemcitabine
GO:0036271 | response to methylphenidate
GO:0036270 | response to diuretic
GO:0035456 | response to interferon beta
GO:0034097 | response to cytokine
GO:0033273 | response to vitamin
GO:0031960 | response to corticosteroid
GO:0031427 | response to methotrexate
GO:0014072 | response to opiate
GO:0010226 | response to lithium ion
GO:0009628 | response to abiotic stimulus
GO:0009615 | response to virus
GO:0009611 | response to wounding
GO:0009415 | response to water stimulus
GO:0009414 | response to water deprivation
GO:0009409 | response to cold
GO:0009408 | response to heat
GO:0009314 | response to radiation
GO:0006970 | response to osmotic stress
GO:0006950 | response to stress
EFO:0600031 | response to benznidazole
EFO:0600030 | response to antiparasitic agent
EFO:0600024 | response to tofacitinib
EFO:0600023 | response to immune checkpoint inhibitor
EFO:0600022 | response to gamma-aminobutyric acid receptor agonists
EFO:0600021 | response to dietary selenium supplementation
EFO:0600017 | response to dolutegravir
EFO:0600013 | response to ropeginterferon alfa-2b
EFO:0011041 | response to BCG intravesical immunotherapy
EFO:0011024 | response to liver transplant@en
EFO:0010971 | response to metamizole@en
EFO:0010969 | response to growth hormone
EFO:0010950 | response to cranial radiation therapy@en
EFO:0010933 | response to dexamethasone@en
EFO:0010825 | response to fluoropyrimidines@en
EFO:0010824 | response to rhododendrol@en
EFO:0010808 | response to tafenoquine@en
EFO:0010757 | response to diet@en
EFO:0010748 | response to zonisamide@en
EFO:0010747 | response to levodopa@en
EFO:0010735 | response to angiotensin receptor blocker@en
EFO:0010731 | response to low calorie diet@en
EFO:0010730 | response to bezlotoxumab@en
EFO:0010718 | response to amisulpride@en
EFO:0010648 | response to vitamin B3@en
EFO:0010577 | response to dimethyl fumarate@en
EFO:0010235 | response to proton-pump inhibitor@en
EFO:0010152 | response to phenylephrine
EFO:0010124 | response to atorvastatin
EFO:0010123 | response to antiviral drug
EFO:0010103 | response to peginterferon alfa-2a
EFO:0010077 | response to belimumab
EFO:0010065 | response to intravenous immunoglobulin therapy
EFO:0010062 | response to salmeterol
EFO:0010055 | response to calcineurin inhibitor
EFO:0010054 | response to mycophenolic acid
EFO:0010053 | response to cyclosporine
EFO:0010052 | response to tacrolimus
EFO:0010051 | response to immunosuppressant
EFO:0009962 | response to radioiodine
EFO:0009958 | response to bisphosphonate
EFO:0009951 | response to surgery
EFO:0009893 | response to oxcarbazepine
EFO:0009796 | response to supplemental oxygen
EFO:0009748 | response to ketamine
EFO:0009467 | response to anti-vascular endothelial growth factor drug
EFO:0009391 | response to tamoxifen
EFO:0009372 | response to ketogenic diet
EFO:0009308 | response to long-chain n-3 PUFA dietary supplementation
EFO:0009279 | response to tenofovir
EFO:0009261 | response to aripiprazole
EFO:0009185 | heart rate response to recovery post exercise
EFO:0009184 | heart rate response to exercise
EFO:0009175 | response to synacthen
EFO:0009170 | response to tyrosine kinase inhibitor
EFO:0009169 | response to deoxygalactonojirimycin
EFO:0009168 | response to prednisolone
EFO:0009167 | response to warfarin
EFO:0009166 | response to ivacaftor - efficacy
EFO:0009131 | response to polyunsaturated fatty acid supplementation
EFO:0008580 | response to taxane
EFO:0008541 | response to opioid
EFO:0008484 | response to carbamazepine
EFO:0008483 | response to trauma exposure
EFO:0008459 | response to mepolizumab
EFO:0008397 | response to disappointment
EFO:0008396 | response to reward
EFO:0008395 | response to darapladib
EFO:0008348 | response to ranibizumab
EFO:0008347 | response to trastuzumab
EFO:0008345 | response to duloxetine
EFO:0008344 | response to placebo
EFO:0008325 | response to sotalol
EFO:0008324 | response to sulfasalazine
EFO:0007981 | response to thiazide
EFO:0007965 | response to combination chemotherapy
EFO:0007943 | response to platinum-based neoadjuvant chemotherapy
EFO:0007925 | response to paliperidone
EFO:0007923 | response to terbinafine
EFO:0007922 | response to sulfonylurea
EFO:0007921 | response to nitrofurantoin
EFO:0007920 | response to fluoroquinolones
EFO:0007919 | response to diclofenac
EFO:0007918 | response to anti-tuberculosis drug
EFO:0007917 | response to tetracyclic antidepressant
EFO:0007916 | response to tricyclic antidepressant
EFO:0007880 | response to dendritic cell-based immunotherapy
EFO:0007871 | response to escitalopram
EFO:0007870 | response to norepinephrine-dopamine reuptake inhibitor
EFO:0007868 | response to erlotinib
EFO:0007867 | response to gefitinib
EFO:0007866 | response to rifampicin
EFO:0007859 | response to interferon
EFO:0007853 | response to mercaptopurine
EFO:0007838 | response to anti-thyroid drug
EFO:0007816 | response to trametinib
EFO:0007815 | response to dabrafenib
EFO:0007809 | response to flupirtine
EFO:0007808 | response to bortezomib
EFO:0007779 | response to pazopanib
EFO:0007768 | response to exercise
EFO:0007767 | response to calcium channel blocker
EFO:0007766 | response to beta blocker
EFO:0007754 | response to immunochemotherapy
EFO:0007684 | response to high fat food intake
EFO:0007683 | response to CAPOX-B
EFO:0007682 | response to cetuximab
EFO:0007676 | response to zileuton
EFO:0007662 | response to triptolide
EFO:0007661 | response to lamotrigine
EFO:0007647 | response to vancomycin
EFO:0007633 | response to thioamide
EFO:0007613 | response to endocrine therapy
EFO:0007612 | response to montelukast
EFO:0007592 | response to bleomycin
EFO:0007044 | response to allogeneic hematopoietic stem cell transplant
EFO:0007043 | response to transplant
EFO:0006998 | response to stavudine
EFO:0006997 | response to cold medicine
EFO:0006996 | response to homoharringtonine
EFO:0006995 | response to diisocyanate
EFO:0006994 | response to gases and fumes exposure
EFO:0006993 | response to mineral dust exposure
EFO:0006992 | response to biological dust exposure
EFO:0006954 | response to triamcinolone acetonide
EFO:0006950 | response to vincristine
EFO:0006916 | response to dalcetrapib
EFO:0006905 | response to abacavir
EFO:0006904 | response to efavirenz
EFO:0006816 | response to heparin
EFO:0006802 | response to red blood cell transfusion
EFO:0006518 | response to losartan
EFO:0006345 | response to phenytoin
EFO:0006337 | response to allopurinol
EFO:0006330 | response to buspirone
EFO:0006329 | response to citalopram
EFO:0006328 | response to venlafaxine
EFO:0006327 | response to sertraline
EFO:0006326 | response to bupropion
EFO:0006325 | response to serotonin-norephinephrine reuptake inhibitor
EFO:0006317 | response to thiopurine
EFO:0006314 | response to high-dose melphalan
EFO:0005944 | cumulative dose response to bevacizumab
EFO:0005943 | response to bevacizumab
EFO:0005853 | response to silica exposure
EFO:0005844 | response to dietary antigen
EFO:0005768 | response to rate control therapy
EFO:0005765 | obsolete_response to haloperidol
EFO:0005658 | response to selective serotonin reuptake inhibitor
EFO:0005657 | response to protease inhibitor
EFO:0005655 | response to cytosine arabinoside
EFO:0005533 | response to non-steroidal anti-inflammatory
EFO:0005526 | response to alcohol
EFO:0005417 | response to mTOR inhibitor
EFO:0005405 | response to antihypertensive drug
EFO:0005404 | response to cold pressor test
EFO:0005403 | response to dietary potassium supplementation
EFO:0005402 | response to low sodium diet
EFO:0005401 | response to high sodium diet
EFO:0005325 | response to angiotensin-converting enzyme inhibitor
EFO:0005260 | response to antimicrotubule agent
EFO:0005257 | response to anthracycline-based chemotherapy
EFO:0005205 | response to dabigatran etexilate
EFO:0005202 | response to hydrochlorothiazide
EFO:0005195 | response to cholinesterase inhibitor
EFO:0005040 | obsolete_response to temozolomide
EFO:0004829 | response to irinotecan
EFO:0004702 | obsolete_response to iloperidone
EFO:0004681 | obsolete_response to lithium
EFO:0004680 | obsolete_response to interferon beta
EFO:0004679 | obsolete_response to glucocorticoid
EFO:0004678 | obsolete_response to statin
EFO:0004677 | obsolete_response to anticonvulsant
EFO:0004676 | obsolete_response to diruetic
EFO:0004675 | obsolete_response to antidepressant
EFO:0004673 | obsolete_response to lapatinib
EFO:0004672 | obsolete_response to gemcitabine
EFO:0004661 | obsolete_response to flurouracil
EFO:0004655 | obsolete_response to methylphenidate
EFO:0004653 | response to TNF antagonist
EFO:0004647 | response to platinum based chemotherapy
EFO:0004645 | response to vaccine

**Comment by krchristie** [comment]
Hi @kallia-p @zoependlington @paolaroncaglia

> I think, given the timing it would be best if we could both approach the 'response to' terms in a systematic way rather than as part of sporadic tickets about specific drugs. If you agree we could start discussing this in a new ticket. If needed I can join one of your future meetings to discuss and plan around the proposed work with regards to the alignment between GO, EFO and OBA 'response to' terms. Please let me know which of the two you prefer and if you think a meeting is warranted which forthcoming Monday we can allocate to this task.

I think a systematic approach would be good. I'll bring this up at the editors meeting next Monday, just as a preliminary discussion to check if other agree with me about a systematic approach at this time. 
 
> I provide a bit of background and context to the preliminary work that I have done thus far.
> I am currently reviewing all of the terms that are in GO and have been imported in EFO (N=75 GO terms, see table below which comprises the EFO created 'response to' specific drug terms as well). I am also examining the OBA cross referenced terms with GO and the overlap between the three ontologies. With regards to EFO some of the imported GO terms are high level grouping terms with the main one being 'response to stimulus' of which 'response to chemical' is a Subclass Of. 'Response to drug' is a Subclass Of 'response to chemical' and I have made a note that you plan to merge this with 'response to xenobiotic'. Further down the GO hierarchy i.e. in the 'response to drug' branch there is some structure incorporated in GO that several EFO terms sit under for example the GO 'response to antipsychotic drug' and GO 'response to antineoplastic agent' have a number of children with either GO or EFO IDs for specific antipsychotic drugs and cancer therapy agents (with one of them being the GO 'response to paclitaxel' which you are planning to obsolete). Small changes like these will have less of an effect to our users but changes in higher level grouping terms will cause much disruption and I wonder why do you think EFO would be a better place for them than GO, and what is the hazard of keeping these kind of terms in GO.
 
> I reviewed the tickets in trying to understand how GO views usage of the 'response to' terms in general.
> In my view an organism's response to an exogenous stimulus whether this is food intake, stress or a pharmacologic intervention constitutes a biological process; variations in our our genes interact with our environment. Many of the GWAS catalog studies associate variants with phenotypes that measure responses. Pharmacogenomics studies will specifically look for variants that modulate efficacy of a drug or adverse/toxic reactions. The GWAS catalog contains associations from gene x environment interactions studies as well. I think it also useful for researchers to know that (a variant affecting expression or function of) gene X which is associated with an adverse reaction to a drug Y (GO 'response to term') is linked to GO biological process Z.

We completely agree that it is useful to know this type of information; we just don't think all of it belongs in GO or should be captured in GO annotations.

There are a couple principles guiding our plans. One is that GO's purpose & scope is to make statements about the normal purpose of gene products. Terms invoking compounds that are only synthetic compounds don't reflect the normal purpose of a gene. 

Another issue is that concepts represented by ChEBI roles like "drug", "vitamin" are context specific, depending on things like the specific usage for a compound that may be a drug in some contexts but not others, or the species for a compound that is a vitamin in some species but not others. For many of these ChEBI roles, we feel that the role is grouping things in ways that are not always true, which is against the principles of an ontology. For "drug" specifically, this is intrinsically out of scope for GO, though we do completely agree that "response to xenobiotic" is a normal process for organisms to deal with their environment. However, within an ontology like GO that encompasses many species, we can not specifiy whether any specific compound is a xenobiotic or not, as it depends on which species you are considering.

I hope that helps explain our thinking, but a more full explanation would probably be best on a call, either a full editors call, or we could set up something smaller on a more flexible schedule.

I look forward to working with you on this.

-Karen

**Comment by kallia-p** [comment]
@krchristie 
Thank you for your reply, looking forward to an update when you discuss this with other editors.
Kallia

**Comment by krchristie** [comment]
Hi @kallia-p @zoependlington @paolaroncaglia

We discussed this at the GO Editors call on 9/27/21. Here's a quick summary:
- GO is in favor of working with EFO in a systematic way to address certain "response to [drug]" terms that we feel do not belong in GO to have EFO take over these terms.
- As we are currently in the middle of another project that is requiring groups to examine and possibly reannotate existing annotations, we will wait to make any obsoletions until that is complete, so we won't start this project for several months.
- Some of the GO terms you are using will stay in GO, even after "response to [drug]" terms are transferred to EFO. Therefore
we propose to create a subset within GO to label terms used by EFO to make sure we let you know about any proposed changes that might affect you.

I've made another ticket to deal with this project, so I'll put all further comments about this project in the new ticket.

Thanks

Karen
